docs/development/language.md

```diff
@@ -0,0 +1,68 @@
+ILIAS Language Handling
+=======================
+ILIAS offers multi-language support for the interface of ILIAS.
```

Comment:
I suggest to explicitly use `user interface` here, because ILIAS also provides other interfaces, e.g. CLI, where multi-language support is not offered.

Reply:
You're right. Accepted.